airflow/utils/operator_helpers.py

```diff
@@ -84,3 +85,24 @@ def context_to_airflow_vars(context, in_env_var_format=False):
         params[AIRFLOW_VAR_NAME_FORMAT_MAPPING['AIRFLOW_CONTEXT_DAG_RUN_ID'][
             name_format]] = dag_run.run_id
     return params
+
+
+def serialize_template_field(template_field):
```

Comment:
Should live in airflow.dag.serialization I think.

Reply:
Moved